q2_fmt/tests/test_engraftment.py

```diff
@@ -645,26 +637,31 @@ def test_mann_whitney_pairwise_against_each_alternative_hypothesis(self):
         for p_val in p_val_total:
             self.assertAlmostEqual(p_val, 1, places=1)
 
-    def test_mann_whitney_reference(self):
-        exp_stats_data = pd.DataFrame({
-            'A:group': ['reference'],
-            'A:n': [5],
-            'A:measure': [10.2488392],
-            'B:group': ['control'],
-            'B:n': [23],
-            'B:measure': [11.6496274],
-            'n': [28],
-            'test-statistic': [37.0],
-            'p-value': [0.23025583],
-            'q-value': [0.23025583],
-        })
-
-        stats_data = mann_whitney_u(distribution=self.faithpd_refdist,
-                                    compare='reference',
-                                    reference_group='reference',
-                                    p_val_approx='asymptotic')
-
-        pd.testing.assert_frame_equal(stats_data, exp_stats_data)
+    # TODO: this test currently fails with the test data because there aren't
```

Comment:
Why was it passing before? In principle, we haven't changed how anything works once you've got a dataframe from whatever artifact you're working with.

Reply:
@ebolyen great question - when we modified the file format, I had to re-create the test data for time/ref dists and ended up creating a dataset that produced a single reference group. I originally thought this would be useful for testing our single ref/control allowance within `group_timepoints` but now that I'm looking back through this I realize that this is a separate issue from what I mentioned in my comment above. We do need multiple reference groups for this test, which is separate from `group_timepoints`. I'm going to re-create the distributions and leave this test as-is!

Reply:
Makes sense, and sounds good, thanks!